docs/packaging/package.yml.md

```diff
@@ -47,20 +47,36 @@ install    : |
 | --------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **name**        | `string`    | The name of the package. This is also used as the base of all sub-package names. Unless unavoidable, this should match the upstream name.                                                                           |
 | **version**     | `string`    | The version of the currently packaged software. This is taken from the tarball in most cases.                                                                                                                       |
 | **release**     | `integer`   | Specifies the current release number. Updates in the package number are based on this `release` number, _not_ the `version` number. As such, to release an update to users, this number must be incremented by one. |
-| **source**      | `dict(s)`   | Upstream source URL (i.e. tarball), with the valid `sha256sum` as a value. Alternatively, the git repository URL prefixed with "git&#124;" and a git tag or commit hash as a value.                                 |
+| **source**      | `dict(s)`   | See [source key](/docs/packaging/package.yml#source-key) below.                                                                                                                                                      |
 | **homepage**    | `string`    | Provides a link to the package's homepage, used in the Software Center.                                                                                                                                             |
 | **license**     | `string(s)` | Valid upstream license(s). Try to ensure these use [SPDX identifiers](https://spdx.org/licenses/).                                                                                                                  |
 | **summary**     | `string`    | Brief package summary, or display name.                                                                                                                                                                             |
 | **component**   | `string`    | Component / group of packages this package belongs to. Check available components via `eopkg lc`.                                                                                                                   |
 | **description** | `string`    | More extensive description of the software, usually taken from the vendor website.                                                                                                                                  |
 
+#### source key
```

Comment:
It feels weird to have a title in full lowercase, but I'm also not sure if it makes sense to capitalize it, given it's in reference to a YAML key... Does anything in the style guide happen to cover this?

Reply:
In my experience, docs follow the case of keys as used (so all lower case in this instance)
Technically speaking, `homepage` does not equal `Homepage` in terms of keys (for example)

I can't find anything specific about this in our style guides, but [the Google style guide](https://developers.google.com/style/capitalization) says

>  Don't use unnecessary capitalization; before you capitalize a word, think about why (and whether) it should be capitalized. 

Also, from a reader's POV - if we refer to a key as `Keyname` but the file expects `keyname`  this would create confusion.

Reply:
backticks. Can't use bold, that doesn't get distinguished from the header
![Capture](https://github.com/getsolus/help-center-docs/assets/23007135/aeb0bb30-cfd6-416b-bf84-a047cf8327b7)

Reply:
or just:


### "source" key

Reply:
I added quotes, just because I think it looks nicer than the inline code formatting